data/human/navy missions intro.txt

```diff
@@ -0,0 +1,1496 @@
+			`Danforth walks back to his chair while you stand there, motionless, and suddenly very fearful of the man in front of you. "I don't want to have this discussion again," he says as he sits down, still calm. "Do I make myself clear?"`
+			`	"Yes, sir," you reply.`
+			`	"Good. Now then..."`
+			label debriefing
+			branch hadley
+				has "lifeDebt"
+			`Danforth pulls out some files from his desk, dropping them on the desktop. "We recently finished training some new recruits. They're all dressed-up and ready to fight, but they've got nowhere to go. They're currently on <stopover>, so we'll bring them here. <stopover> also has some extra defensive guns that they're willing to lend us, with the expectation that we will return them once Geminus has been rebuilt to its former capacity." He hands you the file, and you open it to see a list of names on it. Likely the names of the new recruits to be sent here.`
+			`	"That's the extent of your mission - get to <stopover> with the auxiliaries, pickup the troops and guns, and get them back here. Is there anything else you need to discuss before you leave?"`
+			`	"No, sir."`
+			`Danforth nods. "You have your orders, Lieutenant - get to it!"`
+				goto end
+			label hadley
+			apply
+				payment 1500000
+			`Danforth pulls out some files from his desk, dropping them on the desktop. "We recently finished training some new recruits. They're all dressed-up and ready to fight, but they've got nowhere to go. They're currently on <stopover>, so we'll bring them here. <stopover> also has some extra defensive guns that they're willing to lend us, with the expectation that we will return them once Geminus has been rebuilt to its former capacity." He hands you the file, and you open it to see a list of names on it. Likely the names of the new recruits to be sent here.`
+			`	"That's the extent of your mission - get to <stopover> with the auxiliaries, pickup the troops and guns, and get them back here. Is there anything else you need to discuss before you leave?"`
+			`	"Actually, sir...there is."`
+			`Danforth raises an eyebrow. "Alright, I'm listening."`
+			`	"I met a dockworker at Lodestone named Lucas Hadley. He says that Hines stopped by about a month ago to pick up some materials for the base at New Iceland - is that true?"`
+			`"Let me see," Danforth says, checking his computer. It takes a moment, but he eventually finds his calendar. "Ah, yes. I had sent him out to pick up some framulators for the base. Though I'm curious, <first>...why do you ask?"`
+			`You glance over your shoulder, even though there's no-one who could possibly see you. "Admiral, Hines apparently *decked* Hadley because the latter offended his twisted sense of honor."`
+			`"What?!?" he exclaims, visibly shocked. "Lieutenant, this is a *very* serious accusation you're making," he warns, gravely. "I hope you have evidence to substantiate your claim."`
```

Comment:
Is he loud or is he grave? This all on one line with no break is like getting whiplash.